variables.tf

```
@@ -289,6 +294,11 @@ variable "private_app_gateway_hc_probe_name" {
   default     = "quortex-private-probe"
 }
 
+variable "private_app_gateway_hc_probe_port" {
+  type        = number
+  description = "(Optional) Custom port which will be used for probing the backend servers. The valid value ranges from 1 to 65535. In case not set, port from HTTP settings will be used. This property is valid for Standard_v2 and WAF_v2 only."
+}
```

Reformulated the description, and the default should be null (unless it won't be optional).
```suggestion
  description = "Custom port which will be used for probing the private backend. The valid value ranges from 1 to 65535. In case not set, port from HTTP settings will be used."
  default     = null
}
```